pages/docs/data-deletion.mdx

```diff
@@ -0,0 +1,74 @@
+---
+description: Delete data from Langfuse
+---
+
+# Data Deletion
+
+<AvailabilityBanner
+  availability={{
+    hobby: "full",
+    pro: "full",
+    team: "full",
+    selfHosted: "full",
+  }}
+/>
+
+There may be use-cases where you want to remove selected data from Langfuse, like erroneously created traces in a development flow, user data for PII, or your whole project.
+In case you want to retain only recent data, you can use our [Data Retention](/docs/data-retention) feature.
+
+You can delete unwanted data from Langfuse by:
+
+- Deleting a single trace;
+- Deleting a batch of traces;
+- Deleting all traces that match a query filter;
+- Deleting a project; or
+- Deleting an organization.
+
+Below, we will walk through each of the options and their guarantees.
+
+## Deleting Traces
```

Comment:
Deletions via API seem to be missing here, do you plan to add them in a subsequent release?

Reply:
Yes, that should be a description of the status quo and API deletions will be a net-new feature.

Reply:
sounds good